### ISSUE TITLE
CDN on V6.1.0 has a package error

### ISSUE THREAD

**Issue by jjf21** [fullcalendar/fullcalendar#7176]
### Reduced Test Case

none

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

No connector (vanilla JS)

### Bug Description

When using FullCalendar with importmap, we use CDN: https://cdn.skypack.dev/fullcalendar
But there is a package error that appeared today on this dependency: https://cdn.skypack.dev/new/@fullcalendar/multimonth@v6.1.0/dist=es2019

Error:
```
/*
 * [Package Error] "@fullcalendar/multimonth@v6.1.0" could not be built. 
 *
 *   [1/5] Verifying package is valid…
 *   [2/5] Installing dependencies from npm…
 *   [3/5] Building package using esinstall…
 *   Running esinstall...
 *   ENOENT: no such file or directory, open '@fullcalendar/multimonth/internal.esm.js'
 *   ENOENT: no such file or directory, open '/tmp/cdn/_sOeAtuf8hKzKSQQVw6TQ/node_modules/@fullcalendar/multimonth/internal.esm.js'
 *
 * How to fix:
 *   - If you believe this to be an error in Skypack, file an issue here: https://github.com/skypackjs/skypack-cdn/issues
 *   - If you believe this to be an issue in the package, share this URL with the package authors to help them debug & fix.
 *   - Use https://skypack.dev/ to find a web-friendly alternative to find another package.
 */

console.warn("[Package Error] \"@fullcalendar/multimonth@v6.1.0\" could not be built. \n[1/5] Verifying package is valid…\n[2/5] Installing dependencies from npm…\n[3/5] Building package using esinstall…\nRunning esinstall...\nENOENT: no such file or directory, open '@fullcalendar/multimonth/internal.esm.js'\nENOENT: no such file or directory, open '/tmp/cdn/_sOeAtuf8hKzKSQQVw6TQ/node_modules/@fullcalendar/multimonth/internal.esm.js'");
throw new Error("[Package Error] \"@fullcalendar/multimonth@v6.1.0\" could not be built. ");
export default null;
```

The CDN can't be loaded correctly.
Could you help please, thanks!

**Comment by tmendesilva** [comment]
Maybe https://github.com/fullcalendar/fullcalendar-workspace has some bug like that, multimonth is not loading.

**Comment by arshaw** [comment]
queued for release 5c71d2e97f031058631a17b7c74fdc909afebd9f

**Comment by arshaw** [comment]
Fixed in [v6.1.1](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.1)

https://cdn.skypack.dev/@fullcalendar/multimonth@v6.1.1
